src/dotnet/commands/dotnet-install/dotnet-install-tool/InstallToolCommandLowLevelErrorConverter.cs

```diff
@@ -0,0 +1,58 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using Microsoft.DotNet.ShellShim;
+using Microsoft.DotNet.ToolPackage;
+
+namespace Microsoft.DotNet.Tools.Install.Tool
+{
+    internal static class InstallToolCommandLowLevelErrorConverter
+    {
+        public static IEnumerable<string> GetUserFacingMessages(Exception ex, PackageId packageId)
+        {
+            string[] userFacingMessages = null;
+            if (ex is ToolPackageException)
+            {
+                userFacingMessages = new[]
+                {
+                    string.Format(
```

Comment:
Likewise, this was just `ex.Message` prior to the refactoring.

Reply:
fixed